test/post.js

```diff
@@ -71,6 +88,19 @@ describe('Post', function(){
     });
   });
 
+  describe('Post date fallbacks', function(){
+    describe('no post_date_gmt or post_date', function(){
+      it("should return a default year 0000-00-00 00:00:00 BC", function() {
+        post_missing_both_post_fields.created_at.should.equal(-2211753600000);
```

Comment:
So, the reason I used `0000-00-00 00:00:00` initially was that I actually saw that being used in Wordpress exports. While it's unclear which is semantically more correct to use when importing that into Ghost (given that it's clearly inaccurate), I agree with you that we should instead just go with whatever is easier to implement. UNIX 0 seems fine.